pkg/sloop/webserver/webfiles_test.go

```diff
@@ -0,0 +1,48 @@
+package webserver
+
+import (
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path"
+	"testing"
+)
+
+const (
+	defaultFileMode = os.FileMode(0755)
+	someContents1   = "contents abcd"
+	filePath        = "webfiles/index.html"
+)
+
+func Test_BindataReadWebfile_True(t *testing.T) {
+	expectedOutput, err := Asset(filePath)
+	assert.Nil(t, err)
+
+	actualOutput, _ := readWebfile(filePath, nil, &afero.Afero{afero.NewMemMapFs()})
+	assert.Equal(t, actualOutput, expectedOutput)
+}
+
+func Test_LocalReadWebfile_True(t *testing.T) {
+	notExpectedOutput, _ := Asset(filePath)
+
+	fs := &afero.Afero{afero.NewMemMapFs()}
+	writeFile(t, fs, filePath, someContents1)
+
+	actualOutput, _ := readWebfile("webfiles/index.html", nil, fs)
+
+	assert.NotEqual(t, notExpectedOutput, actualOutput)
+
+}
+
+func Test_FilenotinReqdFormat_False(t *testing.T) {
+	filePath := "index.html"
+	_, err := readWebfile(filePath, nil, &afero.Afero{afero.NewMemMapFs()})
+	assert.Errorf(t, err, "Webfile %v is invalid.  Must start with %v", filePath, prefix)
```

Comment:
if you're going to verify the contents of the error, consider moving the format string used to generate the error to be a named constant, then reuse the constant here